db/migrate/20191230121237_clean_up_accrediting_provider_data.rb

```diff
@@ -0,0 +1,14 @@
+class CleanUpAccreditingProviderData < ActiveRecord::Migration[6.0]
```

Comment:
This PR looks good overall but I'm not so sure about using migrations what are effectively 'data changes' rather than 'schema changes'. I think a temporary rake task is the best place for this logic ...

Reply:
We've previously used migrations for data changes, but i don't have any strong feelings on this. Perhaps @misaka  can weigh in.

Reply:
I agree that this doesn't warrant a data migration. I think we only do data migrations when necessary, this one doesn't qualify because it doesn't have to be in lock-step with a schema change or code change.

Reply:
Ok. As discussed i've edited the provider edit command 👍